docs/structured-streaming-kafka-integration.md

```diff
@@ -441,7 +441,7 @@ Apache Kafka only supports at least once write semantics. Consequently, when wri
 or Batch Queries---to Kafka, some records may be duplicated; this can happen, for example, if Kafka needs
 to retry a message that was not acknowledged by a Broker, even though that Broker received and wrote the message record.
 Structured Streaming cannot prevent such duplicates from occurring due to these Kafka write semantics. However,
-if writing the query is successful, then you can assume that the query output was written at least once. A possible
+if writing the query is successful, then you can assume that the query output was written exactly once. A possible
```

Comment:
Hi, @wenxuanguan . This looks wrong in this context. Could you explain your thought?

Reply:
Thanks for your reply.
I thought this describes the situation that query writes to kafka successfully and no records are duplicated. How about change to `So if writing the query is successful, then you can assume that the query output was written at least once`, which will not be confused by `However`

Reply:
The existing one looks okay to me. Let's remove this doc change from this PR.

Reply:
@dongjoon-hyun Thanks, done